### PR TITLE
Closes #2487 general issue move deprecated arguments and functions to next phase or remove

### LINKED ISSUE #2487: General Issue: Move deprecated arguments and functions to next phase or remove
### Background Information

These function arguments should be further deprecated or removed

image

Please follow this guidance when deprecating https://pharmaverse.github.io/admiraldev/dev/articles/programming_strategy.html#deprecation

### Definition of Done

- [ ] Functions or arguments being removed should be documented in the changelog
- [ ] Function or arguments being removed based on their phase of deprecation should also be documented
  - [ ] Bonus points if you split them up as Phase 1, 2, 3 so easier to track!
- [ ] Be sure functions/argument documents are appropriately updated and/or removed
- [ ] Be sure unit tests appropriately capture deprecation where appropriate
- [ ] Remove any old deprecation unit tests no longer relevant

@pharmaverse/admiral @pharmaverse/admiral_comm this is a good one to knock out!!  Very helpful to us to do this in the beginning of the cycle!

### PR DESCRIPTION
**Pull request by StefanThoma**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Review the [Cheat Sheet](https://github.com/pharmaverse/admiral/blob/main/inst/cheatsheet/admiral_cheatsheet.pdf). Make any required updates to it by editing the file `inst/cheatsheet/admiral_cheatsheet.pptx` and re-upload a PDF and a PNG version of it to the same folder. (The PNG version can be created by taking a screenshot of the PDF version.)
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers). A Developer Notes section is available in `NEWS.md` for tracking developer-facing issues.  
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by StefanThoma** [comment]
It complains about the link: 
https://www.fda.gov/media/116737/download
But for me it works...

**Comment by StefanThoma** [comment]
I see, thi

> It complains about the link: https://www.fda.gov/media/116737/download But for me it works...

Ah this has been discussed here: https://github.com/pharmaverse/admiral/pull/2494#issuecomment-2311789386

**Comment by StefanThoma** [comment]
@bundfussr all deprecations went from `error` to `removal`. 
There were no deprecations in stage 1.

File: NEWS.md [hunk 1]
```diff
@@ -8,6 +8,28 @@
...
+  
+- The following function arguments are entering the next phase of the deprecation process: (#2487)
+
+### Phase 1 (warning)
+
+### Phase 2 (error)
+
+### Phase 3 (removed)
+  - `consolidate_metadata(check_keys)`
+  - Removed at v1.1.1 `compute_egfr(wt)`
```
Comment (bms63) [inline NEWS.md:20]: Did this one get updated??  I don't see it changed in the list of files.   `compute_kidney.R`

Comment (bms63) [reply NEWS.md:20]: Or did we miss it in the last release?

Comment (StefanThoma) [reply NEWS.md:20]: In the repo: https://github.com/pharmaverse/admiral/blob/2487-general-issue-move-deprecated-arguments-and-functions-to-next-phase-or-remove/R/compute_kidney.R
There is no `wt` argument anymore. 
This is the case for a few other arguments down the line of the list in news.md (indicated by me with: `Removed at v1.1.1`)

Comment (bms63) [reply NEWS.md:20]: Ah so we missed these!  Thanks for clarifying and be specific in the news section.

**Review by bms63 — APPROVED** [review]
Whoa - feels like end of era where we are not constantly deprecating stuff!  @rossfarrugia no phase 1 or phase 2 happening...we at least not yet but it will be minimal.

**Comment by bms63** [comment]
@StefanThoma do you have time to check on admiraldev or ask @sadchla-codes  where she is at on finishing.  I think the updates were pretty minimal.

**Comment by bms63** [comment]
Many thanks @StefanThoma for getting this completed!

**Pull request merged**